conftest.py

```diff
@@ -65,3 +68,20 @@ def _container(registry):
 @pytest.fixture(name="close_me")
 def _close_me():
     return CloseMe()
+
+
+@pytest.fixture(name="create_module")
+def _create_module(tmp_path):
```

Comment:
If you use this fixture only in test_registry, it should go to test_registry. I'm trying to keep the fixture namespace as clean as possible.